src/malli/generator.cljc

```diff
@@ -454,6 +454,20 @@
                         {:infinite? (get props :gen/infinite? false)
                          :NaN? (get props :gen/NaN? false)})
                       (-min-max schema options))))
+(defmethod -schema-generator :float [schema options]
+  (let [max-float #?(:clj Float/MAX_VALUE :cljs (.-MAX_VALUE js/Number))
+        min-float #?(:clj Float/MIN_VALUE :cljs (.-MIN_VALUE js/Number))
```

Comment:
Ah yes good catch 🙏

Reply:
I just meant for JVM, `(.-MIN_VALUE js/Number)` seems appropriate for cljs.

Reply:
Do we want the `min-float` to be a negative number? Because I think `(.-MIN_VALUE js/Number)` would be [the smallest positive number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_VALUE#description).